mne/filter.py

```diff
@@ -2515,6 +2515,8 @@ def savgol_filter(self, h_freq, verbose=None):
 
         .. versionadded:: 0.9.0
 
+        When working on SourceEstimates the sample rate of the original data is inferred from tstep.
+
```

Comment:
Versionadded always goes at the bottom
```suggestion
        When working on SourceEstimates the sample rate of the original data is inferred from tstep.

        .. versionadded:: 0.9.0
```